app/src/main/java/com/nutomic/syncthingandroid/activities/SettingsActivity.java

```diff
@@ -311,8 +315,8 @@ public void onServiceConnected() {
         public void onServiceStateChange(SyncthingService.State currentState) {
             mApi = mSyncthingService.getApi();
             boolean isSyncthingRunning = (mApi != null) &&
-                        mApi.isConfigLoaded() &&
-                        (currentState == SyncthingService.State.ACTIVE);
+                    mApi.isConfigLoaded() &&
+                    (currentState == SyncthingService.State.ACTIVE);
```

Comment:
I'd suggest keeping the diff as small as possible here as there's no change.

Reply:
I honestly don't even know what happened here. This change doesn't even look better. I'll revert.

Reply:
If you're using android studio it may be auto reformat code being turned on.